app/uk/gov/hmrc/nisp/views/account_mqp.scala.html

```diff
@@ -46,73 +46,47 @@
 @defining(Some(user)) { implicit userOption =>
 @nispMain(userLoggedIn = true, browserTitle = Messages("nisp.main.title"), analyticsAdditionalJs = Some(analyticsAdditionalJs)) {
 
-<div class="grid services">
-    <h1 class="heading-large title-sp">@Messages("nisp.main.description.mqp")<span style="white-space:nowrap"> @Dates.formatDate(spSummary.statePensionAge.date.localDate)</span>, @Messages("nisp.main.whenYoullBe") @spSummary.statePensionAge.age</h1>
-    <div class="panel-indent mqp">
-        <p><strong>
-           @if(spSummary.numberOfQualifyingYears == 1) {
-               @Messages("nisp.mqp.youcurrentlyhave.singular", spSummary.numberOfQualifyingYears, spSummary.lastProcessedDate.localDate.getYear().toString())
-           } else {
-               @Messages("nisp.mqp.youcurrentlyhave.plural", spSummary.numberOfQualifyingYears, spSummary.lastProcessedDate.localDate.getYear().toString())
-           }
-        </strong></p>
+    <h2 class="heading-large">@Messages("nisp.main.h1.title")</h2>
+    <h3 class="heading-medium">@Messages("nisp.main.description.mqp") @Dates.formatDate(spSummary.statePensionAge.date.localDate), @Messages("nisp.main.whenYoullBe") @spSummary.statePensionAge.age</h3>
+    <div class="panel-indent">
+        <p> @Messages("nisp.mqp.youcurrentlyhaveUsing", Time.years(spSummary.numberOfQualifyingYears), spSummary.lastProcessedDate.localDate.getYear().toString())</p>
     </div>
 
     @if(canGetPension) {
-        <h2 class="heading-medium">@Messages("nisp.mqp.possible")</h2>
-    } else {
-        <h2 class="heading-medium">@Messages("nisp.mqp.notpossible")</h2>
-    }
-
-    <p>
-        @Messages("nisp.mqp.youneedtoadd",
-                  Time.years(numberOfYearsMissing),
-                  Time.years(spSummary.yearsToContributeUntilPensionAge),
-                  Dates.formatDate(spSummary.statePensionAge.date.localDate))
-
-        @if(canGetPension == false) {
-            @Messages("nisp.mqp.afterthis")
+        <h3 class="heading-medium">@Messages("nisp.mqp.possible")</h3>
+        <p>  @Messages("nisp.mqp.youneedtoadd",
+            Time.years(numberOfYearsMissing),
+            Time.years(spSummary.yearsToContributeUntilPensionAge),
+            Dates.formatDate(spSummary.statePensionAge.date.localDate)) </p>       
+        @if(spSummary.numberOfQualifyingYears + spSummary.numberOfGapsPayable>=Constants.minimumQualifyingYearsNSP) {
```

Comment:
This if statement seems a bit redundant as they will always have gaps to fill, based on the logic used to calculate `canGetPension`